ambari-server/src/main/java/org/apache/ambari/server/api/resources/CompatibleRepositoryVersionDefinition.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The double *'s is a bit different.  Does this worth with the rat check?

Reply:
My IntelliJ does this but rat checker doesn't really care.